inspirehep/modules/workflows/tasks/refextract.py

```diff
@@ -28,6 +28,7 @@
 
 from refextract import extract_journal_reference, extract_references_from_file
 
+from inspirehep.config import REFEXTRACT_JOURNAL_KB_PATH
```

Comment:
You should never do `from inspirehep.config import ...`: this prevents this path from being overridable on production. When you have to import from the config you need to have an application context and access `current_app.config`